v1-api-conversion/src/main/java/io/strimzi/kafka/api/conversion/v1/converter/conversions/NodePoolConversions.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.api.conversion.v1.converter.conversions;
+
+import io.strimzi.api.kafka.model.kafka.JbodStorage;
+import io.strimzi.api.kafka.model.kafka.PersistentClaimStorage;
+import io.strimzi.api.kafka.model.kafka.SingleVolumeStorage;
+import io.strimzi.api.kafka.model.kafka.Storage;
+import io.strimzi.api.kafka.model.nodepool.KafkaNodePool;
+import io.strimzi.kafka.api.conversion.v1.converter.ApiConversionFailedException;
+
+/**
+ * Class for holding the various conversions specific for the KafkaNodePool API conversion
```

Comment:
Just a nit but I noticed that some classes use a generic Javadoc like this "Class for holding the various conversions specific for the v1 API conversion" while others (like the node pool one) specifies the custom resource. I would pick one but I am fine with both.